lib/token.js

```diff
@@ -212,16 +248,22 @@ function getDefaultOAuthParams(sdk, oauthOptions) {
   oauthOptions = util.clone(oauthOptions) || {};
 
   var defaults = {
+    grantType: 'implicit',
     clientId: sdk.options.clientId,
     redirectUri: sdk.options.redirectUri || window.location.href,
     responseType: 'id_token',
-    responseMode: 'okta_post_message',
+    responseMode: 'fragment',
     state: util.genRandomString(64),
     nonce: util.genRandomString(64),
     scopes: ['openid', 'email'],
     ignoreSignature: sdk.options.ignoreSignature
   };
   util.extend(defaults, oauthOptions);
```

Comment:
IWhat happens when a refresh token is requested using the scope `offline_access`? Does Okta return an error?

Reply:
I will test this

Reply:
test added